### DISCUSSION TITLE
Where did the list of APIs and corresponding project folders go?

### DISCUSSION THREAD

**Discussion by 163a** [Dynatrace/dynatrace-configuration-as-code#417] [Q&A] [answered]
Hi, listed on the new docs page
https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/Guides/add_new_api
it references "table of supported APIs"

It seems this table disappeared.
How can I know folder structure I need to create in my project to map to the different APIs?

**Comment by 163a** [comment, marked as answer]
Found the raw file: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/pkg/api/api.go

**Reply by UnseenWizzard** [reply]
Hi @163a , the table still exists here: https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/configuration/configTypes_tokenPermissions/

We seem to call it "supported APIs" as well as "configuration types"  inconsistently it seems, sorry for that

**Comment by interceptor** [comment]
Related: If we could consider my improvement request (see post "Adding a new *custom* API") that would be great!
